src/geometry/euclidean/sphere/basic.lean

```diff
@@ -132,16 +135,9 @@ begin
   exact ⟨c, r, λ p hp, hcr p (hs hp)⟩
 end
 
-include V
-
 /-- The empty set is cospherical. -/
-lemma cospherical_empty : cospherical (∅ : set P) :=
-begin
-  use add_torsor.nonempty.some,
-  simp,
-end
-
-omit V
+lemma cospherical_empty [nonempty P] : cospherical (∅ : set P) :=
```

Comment:
This lemma has changed, and now assumes `[nonempty P]` instead of `[inner_product_space ℝ V] [normed_add_torsor V P]`